eiffel-vocabulary/EiffelConfidenceLevelModifiedEvent.md

```diff
@@ -84,6 +84,12 @@ __Legal targets:__ [EiffelConfidenceLevelModifiedEvent](../eiffel-vocabulary/Eif
 __Multiple allowed:__ Yes  
 __Description:__ Used in events summarizing multiple confidence levels. Example use case: the confidence level "allTestsOk" summarizes the confidence levels "unitTestsOk, "scenarioTestsOk" and "deploymentTestsOk", and consequently links to them via __SUB_CONFIDENCE_LEVEL__. This is intended for purely descriptive, rather than prescriptive, use.
 
+### CONFIDENCE_BASIS
+__Required:__ No  
+__Legal targets:__ [EiffelTestCaseTriggeredEvent](../eiffel-vocabulary/EiffelTestCaseTriggeredEvent.md), [EiffelTestSuiteStartedEvent](../eiffel-vocabulary/EiffelTestSuiteStartedEvent.md)  
```

Comment:
Shouldn't we base our confidence on the actual results of the test case and/or test suite instead? I.e. Shouldn't legal targets be TestCaseFinished and TestSuiteFinished?

Reply:
I'll take that question with me when we discuss the PR.

I have assumed that we always should point to the triggered event but let's hear what the others think.

Reply:
What does "when we duscuss the PR" mean? Will you bring it to a community meeting? If so, when? :)

Reply:
We will discuss this on the 31 of August during the community meeting.

Reply:
@t-persson In [f3af483](https://github.com/eiffel-community/eiffel/pull/368/commits/f3af483bc6575bc922b908df2026472a0b8e0be0) I have changed to link to the finished events instead.